src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/config/ApplicationConfig.kt

```diff
@@ -102,4 +110,20 @@ class ApplicationConfig {
                 .setReadTimeout(Duration.of(2, ChronoUnit.SECONDS)),
         )
     }
+
+    @Bean
+    fun prosesseringInfoProvider() = object : ProsesseringInfoProvider {
+
+        override fun hentBrukernavn(): String = try {
+            SpringTokenValidationContextHolder().tokenValidationContext.getClaims("azuread")
+                .getStringClaim("preferred_username")
+        } catch (e: Exception) {
+            throw e
+        }
+
+        override fun harTilgang(): Boolean {
+            return kallKommerFraProssesering()
```

Comment:
👍 
Vi har lagt inn sjekk på rolle 
https://github.com/navikt/tilleggsstonader-sak/pull/26/files#diff-2b52a396904ffbbaedb841dba036d98544d96e4b9702589afbfae8f29d3d3c1bR28

Reply:
Ja - var litt innom rolle-sjekk først, men hadde ikke så lyst til å legge inn "erProd" sjekk (siden vi ikke har rolle for dette i dag i preprod. ) Tenkte vi kunne kreve saksbehandlerrolle-ef i preprod, men vi deler app med BA, så det kunne blitt litt rødt i UI for noen idabrukere da.

Reply:
Vi endrer til rollesjekk 👍